src/utils/AppConsts.js

```diff
@@ -12,20 +12,20 @@ const PROD = 'http://api.loom.digital';
 const JSON = 'json';
 const CSV = 'csv';
 
-const HIDDEN = 'hidden';
-const DISCOVER = 'discover';
-const READ = 'read';
-const WRITE = 'write';
-const OWNER = 'owner';
+const HIDDEN = 'HIDDEN';
```

Comment:
you're using `AppConsts.js` to hold absolutely all constants. I think it would be better to have a bit more separation. for example, all of these seem to be permissions related constants, so why not have `PermissionsConsts`, or `RolesConsts`, etc...?